src/dispatch/static/dispatch/src/incident/TableFilterDialog.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <v-dialog v-model="display" max-width="600px">
+    <template v-slot:activator="{ on }">
+      <v-badge :value="numFilters" bordered overlap :content="numFilters">
+        <v-btn color="secondary" dark v-on="on">Filter Columns</v-btn>
+      </v-badge>
+    </template>
+    <v-card>
+      <v-card-title>
+        <span class="headline">Column Filters</span>
+      </v-card-title>
+      <v-list dense>
+        <!--
+          <v-list-item>
+            <v-list-item-content>
+              <individual-combobox v-model="commander" label="Commanders"></individual-combobox>
+            </v-list-item-content>
+          </v-list-item>
+          <v-list-item>
+            <v-list-item-content>
+              <individual-combobox v-model="reporter" label="Reporters"></individual-combobox>
+            </v-list-item-content>
+          </v-list-item>
+          -->
```

Comment:
Are we planning to use this in the future? If not, we should remove it.

Reply:
We will as soon as #124 is merged.